mahiru/components/domain_administrator.py

```diff
@@ -444,30 +433,15 @@ def _remove_containers(self, containers: Iterable[Container]) -> None:
             containers: List of containers to remove.
         """
         for container in containers:
+            try:
+                container.stop()
+            except Exception:
+                # ignore, we may be cleaning up an error situation
+                pass
+
             try:
                 container.remove()
             except Exception as e:
                 logger.warning(
                         f'Failed to remove container {container.id}: {e}')
```

Comment:
This warning might be deceiving. Because we use the force in the next step to remove it anyway. Would maybe be better to say: 'Failed to remove container safely, removing with force now'

Reply:
Good point!